infrastructure/infrastructure.tf

```diff
@@ -79,8 +79,29 @@ module "elastic_resources" {
   ssm_source_stage = "${local.ssm_source_stage}"
 }
 
+module "nmap_task_shared" {
```

Comment:
I really don't like this shared name. We are not sharing this module. nmap_task_resources?